docs/model_objective/index.md

```diff
@@ -1,6 +1,6 @@
 # Model Overview
 
-This section describes:
+This section provides a general introduction about the ETSAP-TIAM model, the time resolution considered and the objective function for the linear optimization model. :
```

Comment:
Please remove a colon at the end of this sentence.

Reply:
Okay will do